inst/gams/scripts/datainput.gms

```diff
@@ -83,10 +93,12 @@ p_feDemand(hs,bs,vin,reg,typ,ttot) =
 ;
 
 * renovation cost (hierarchical renovation)
+$ifThenE.noTotal (sameas("%SEQUENTIALREN%","TRUE"))
 p_specCostRen(cost,state,bsr,hsr,vin,subs,ttot) =
     p_specCostRenBS(cost,state,bsr,vin,subs,ttot)
   + p_specCostRenHS(cost,state,hsr,vin,subs,ttot)
 ;
+$endIf.noTotal
```

Comment:
I'd prefer if we named conditions consistently. Also I like it better to use the condition as a name and not what it controls. But that's just personal preference.
```suggestion
$ifThenE.sequentialRen (sameas("%SEQUENTIALREN%","TRUE"))
p_specCostRen(cost,state,bsr,hsr,vin,subs,ttot) =
    p_specCostRenBS(cost,state,bsr,vin,subs,ttot)
  + p_specCostRenHS(cost,state,hsr,vin,subs,ttot)
;
$endIf.sequentialRen 
```

Reply:
Ah, okay. Then you can also get rid of the more complex `$ifThenE` and replace if with `$ifThen`.